src/electron/platform/android/rule-information-provider.ts

```diff
@@ -117,13 +120,25 @@ export class RuleInformationProvider {
         };
     }
 
-    private includeColorContrastResult = (ruleResultsData: RuleResultsData): boolean => {
-        return false; // Temporarily disable color contrast rule
-        // return ruleResultsData.props['Confidence in Color Detection'] === 'High';
+    private getColorContrastResultStatus = (
+        ruleResultsData: RuleResultsData,
+    ): InstanceResultStatus => {
+        if (
+            ruleResultsData.status == 'FAIL' &&
+            ruleResultsData.props['Confidence in Color Detection'] === 'High'
+        )
+            return 'unknown';
+
+        return 'pass';
     };
 
-    private includeAllResults = (ruleResultsData: RuleResultsData): boolean => {
-        return true;
+    private getStandardResultStatus = (ruleResultsData: RuleResultsData): InstanceResultStatus => {
+        switch (ruleResultsData.status) {
+            case 'FAIL':
+                return 'fail';
+            default:
+                return 'pass';
```

Comment:
The previous behavior from `scan-results-to-unified-results` treated non-PASS non-FAIL as unknown, rather than pass, which sounds more reasonable to me; why the behavior change?

Reply:
Unknown results are displayed in the Needs Review tab, and we only want to display Color Contrast results in the Needs Review tab. To accomplish the desired behavior, we have two options:
1. Only mark the appropriate CC results as unknown
2. Update the Needs Review display behavior to display results based on something other than status unknown

I can see the value of option 2, but choosing it raises the question in my mind: what is the practical meaning of status unknown if not to indicate that a result needs review? I couldn't come up with a good answer to that question, so I went with option 1. My understanding of the values of `InstanceResultStatus` is:
- `fail`: we believe the result indicates an accessibility issue, so we display it in Automated Checks
- `unknown`: we believe the result is likely to indicate an accessibility issue, so we display it in Needs Review
- `pass`: we do not believe the result indicates an accessibility issue, so we do not display it

I am certainly open to another approach if I'm missing something about what unknown should indicate!

Reply:
That's fair enough; assuming PM confirms that we do want to suppress unknown cases for other rules, I'd be okay with sticking with this. It might be nice to add some comments on the definition of the `InstanceResultStatus` type to clarify the intended meaning (particularly, that "pass" may include results which are very low-confidence failures, in addition to high-confidence non-failures)

Reply:
Added comment as suggested!